### PR TITLE
TT1 blocks: cleanup CSS

### LINKED ISSUE #24: TT1 blocks: Remove CSS that is no longer needed
Some of the CSS on tt1 blocks can now be replaced on theme.json or with block attributes on the markup, let's clean up the stylesheets.

### PR DESCRIPTION
**Pull request by MaggieCabrera**
Closes #24

This PR is a WIP. We are aiming to remove most of (if not all) the CSS from the theme.

### CODE REVIEW

File: tt1-blocks/assets/css/blocks.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /*--------------------------------------------------------------
-# Cover
+# Border, for backwards compatibility
 --------------------------------------------------------------*/
 
-.wp-block-cover.is-style-twentytwentyone-border {
+.is-style-twentytwentyone-border {
```
Comment (MaggieCabrera) [inline tt1-blocks/assets/css/blocks.css:5]: this is needed so we don't lose the border on already inserted patterns with the old markup. I hope by the end of this PR we can move this to style.css

File: tt1-blocks/assets/css/style-shared.css [hunk 2]
```diff
@@ -23,49 +23,3 @@ a:hover {
...
-
-/*
- * Gutenberg remotes this underline, but Twenty Twenty-One uses it. 
- */
-.site-header h1.wp-block-site-title a:not(:hover):not(:focus):not(:active) {
-	text-decoration: underline;
-}
-
-/*
- * Alignment styles.
```
Comment (MaggieCabrera) [inline tt1-blocks/assets/css/style-shared.css:35]: We will add the theme.json for the global padding on [this issue](https://github.com/WordPress/community-themes/issues/25#issuecomment-1490110408)

**Comment by carolinan** [comment]
Should we merge this and the theme.json update to one branch (I don't know how but I think it would be easier)

**Comment by MaggieCabrera** [comment]
> Should we merge this and the theme.json update to one branch (I don't know how but I think it would be easier)

I don't, either. We could make one PR based on the other but I don't know how much we need to do that, this PR so far is not touching on theme.json, I was expecting the theme.json PR to be merged before this one anyway

**Comment by carolinan** [comment]
I'm finding it difficult to see how the theme.json changes affect the theme because that PR doesn't remove the CSS.

**Comment by MaggieCabrera** [comment]
> I'm finding it difficult to see how the theme.json changes affect the theme because that PR doesn't remove the CSS.

I'm not working on this today, if you want you can cherry pick my commits to your PR and we can close this one

**Comment by MaggieCabrera** [comment]
Merged these changes inside https://github.com/WordPress/community-themes/pull/29 so we just work on that one branch. Closing this.

**Pull request closed without merging**